xilinx/vc707/constraints/vc707-master.xdc

```diff
@@ -1,3 +1,6 @@
 #-------------- MCS Generation ----------------------
 set_property CFGBVS GND                               [current_design]
 set_property CONFIG_VOLTAGE 1.8                       [current_design]
+
+set_property EXTRACT_ENABLE YES                       [get_cells dut_/spi_0_1/mac/phy/txd_reg*]
```

Comment:
what does this do? do you need this if you keep the synchronizer above ?

Reply:
This is independent of the sync stages - there was a problem that we ( I and Henry S.)  saw with respect to the reset signal on the txd reg, hence we explicitly ensured it is pulled out.